cmd/forwarder-generator/main.go

```diff
@@ -0,0 +1,83 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	yaml "gopkg.in/yaml.v2"
+
+	logging "github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1"
+	"github.com/openshift/cluster-logging-operator/pkg/generators/forwarding"
+	"github.com/openshift/cluster-logging-operator/pkg/k8shandler"
+	"github.com/openshift/cluster-logging-operator/pkg/logger"
+	"github.com/sirupsen/logrus"
+)
+
+
+// HACK - This command is for development use only
+func main() {
+	if logLevel := os.Getenv("LOG_LEVEL"); logLevel != "" {
+		level, err := logrus.ParseLevel(logLevel)
+		if err != nil {
+			logrus.Errorf("Unable to evaluate the LOG_LEVEL: %s %v", logLevel, err)
+			os.Exit(1)
+		}
+		logrus.SetLevel(level)
+	}
+
+	if len(os.Args) < 2 {
+		logrus.Error("Need to pass the logging forwarding yaml as an arg")
+		os.Exit(1)
+	}
```

Comment:
A good option for flags is the already included `github.com/spf13/pflag` as in the operator main.go